igvc_navigation/launch/octomap.yaml

```diff
@@ -0,0 +1,76 @@
+octree:
+    resolution: 0.2 # width & length of each grid cell in m
+    clamping:
+        max: 0.95 # Value of octree considered 100% occupied stable
+        min: 0.05 # Value of octree considered 0% occupied stable
+# P(x | x) where x is either miss or hit. So prob_miss: 0.2 means that I'm 20% confident a miss is a miss
+probability_model:
+    lidar:
+        scan:
+            prob_miss: 0.55
+            prob_hit: 0.9
+        ground:
+            prob_miss: 0.6
+            prob_hit: 0.6 # Not used
+        free_space:
```

Comment:
how is free_space different from scan?

Reply:
I've included comments to differentiate them. prob_miss in scan refers the the ray between the sensor and the occupied points, while free_space refers to the rays for angles which the lidar doesn't register any occupied points at all.

Reply:
That seems just like a regular miss to me.